qfdmo/models/acteur.py

```diff
@@ -589,13 +589,18 @@ class Meta:
         validators=[clean_parent],
     )
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._original_parent = self.parent
```

Comment:
Sans juger du scope de `__init__` et `_original_parent` par précaution j'aurais mis `self._original_parent = getattr(self,"_original_parent",self.parent)` qui parait plus robuste quel que soit le cas de figure.

Reply:
```suggestion
     if not self._original_parent:
        self._original_parent = self.parent
```

peut-être plus clair comme ça ?

Reply:
ou alors

```suggestion
        self._original_parent = self._original_parent or self.parent
```